utils/index.js

```diff
@@ -288,9 +297,38 @@ const getPreviousRoute = (req, routes = defaultRoutes) => {
     return oneRouteBack
   }
 
+  if(req.query && req.query.ref) {
+    //I'm assuming here we want to let them go back to yes/no
+    if( req.path.includes('amount') ) {
```

Comment:
This works in some cases, not in others. I think it's a reasonable guess though.

So, if you do
- Did you pay rent? [Review]
- Yes (I want to go back to the "deduct your rent" page)
- Back

Then it makes sense. But if you do:

- Rent paid? [Review]
- Go back (indenting to go back to the check answers page)
- Did you pay rent?

Then maybe it's not what they expected. 

Anyway, it's kind of weird but it seems like it's reasonable. Normally, I would default to "don't add more if checks for edge cases" but it's okay as is.

Reply:
I can get rid of it, no problem